package.json

```diff
@@ -46,6 +46,7 @@
 		"@ardenthq/arkvault-url": "^1.2.0",
 		"@ardenthq/sdk": "2.0.0-alpha.66",
 		"@ardenthq/sdk-ark": "2.0.0-alpha.66",
+		"@ardenthq/sdk-cryptography": "2.0.0-alpha.66",
```

Comment:
can we avoid this? we already have parts of the crypto sdk moved into vault so it will be better to use those or add the missing features to it